### ISSUE TITLE
Google Colab: "Model was trained with… Bad things might happen unless you revert to…"

### ISSUE THREAD

**Issue by 7k50** [m-bain/whisperX#258]
Seeing the warnings in Google Colab:

```
Model was trained with pyannote.audio 0.0.1, yours is 2.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.0.0+cu118. Bad things might happen unless you revert torch to 1.x.
Model was trained with pyannote.audio 0.0.1, yours is 2.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.0.0+cu118. Bad things might happen unless you revert torch to 1.x.
```

These appear incidentally as I am having trouble transcribing & diarizing a wav file (these processes seem to complete as indicated below, but an `.srt` file is never created even though `--output_format "srt"` and `--output_dir` are specified). I'm just curious if these warnings may be pertinent for the problem I'm facing, or if they can be ignored in general:

```
Downloading model.bin: 100% 3.09G/3.09G [00:47<00:00, 65.6MB/s]
100%|█████████████████████████████████████| 16.9M/16.9M [00:01<00:00, 15.4MiB/s]
Lightning automatically upgraded your loaded checkpoint from v1.5.4 to v2.0.2. To apply the upgrade to your files permanently, run `python -m pytorch_lightning.utilities.upgrade_checkpoint --file ../root/.cache/torch/whisperx-vad-segmentation.bin`
Model was trained with pyannote.audio 0.0.1, yours is 2.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.0.0+cu118. Bad things might happen unless you revert torch to 1.x.
>>Performing transcription...
Downloading: "https://download.pytorch.org/torchaudio/models/wav2vec2_fairseq_large_lv60k_asr_ls960.pth" to /root/.cache/torch/hub/checkpoints/wav2vec2_fairseq_large_lv60k_asr_ls960.pth
100% 1.18G/1.18G [00:09<00:00, 133MB/s]
>>Performing alignment...
>>Performing diarization...
Downloading (…)olve/2.1/config.yaml: 100% 500/500 [00:00<00:00, 3.04MB/s]
Downloading pytorch_model.bin: 100% 17.7M/17.7M [00:00<00:00, 34.5MB/s]
Downloading (…)/2022.07/config.yaml: 100% 318/318 [00:00<00:00, 2.28MB/s]
Lightning automatically upgraded your loaded checkpoint from v1.5.4 to v2.0.2. To apply the upgrade to your files permanently, run `python -m pytorch_lightning.utilities.upgrade_checkpoint --file ../root/.cache/torch/pyannote/models--pyannote--segmentation/snapshots/c4c8ceafcbb3a7a280c2d357aee9fbc9b0be7f9b/pytorch_model.bin`
Model was trained with pyannote.audio 0.0.1, yours is 2.1.1. Bad things might happen unless you revert pyannote.audio to 0.x.
Model was trained with torch 1.10.0+cu102, yours is 2.0.0+cu118. Bad things might happen unless you revert torch to 1.x.
Downloading (…)ain/hyperparams.yaml: 100% 1.92k/1.92k [00:00<00:00, 14.8MB/s]
Downloading embedding_model.ckpt: 100% 83.3M/83.3M [00:00<00:00, 168MB/s]
Downloading (…)an_var_norm_emb.ckpt: 100% 1.92k/1.92k [00:00<00:00, 14.0MB/s]
Downloading classifier.ckpt: 100% 5.53M/5.53M [00:00<00:00, 234MB/s]
Downloading (…)in/label_encoder.txt: 100% 129k/129k [00:00<00:00, 6.73MB/s]
^C
```

How I install WhisperX in Google Colab currently:

```
!pip install git+https://github.com/m-bain/whisperx.git
!pip install light-the-torch

#conda install pytorch==2.0.0 torchvision==0.15.0 torchaudio==2.0.1 pytorch-cuda=11.8 -c pytorch -c nvidia
!ltt install --pytorch-computation-backend=cu118 torch==2.0.0 torchvision==0.15.0 torchaudio==2.0.1
```

I run WhisperX via:

```
!whisperx "{audio_path_wav}" \
    --task "transcribe" \
    --model "{model_name}" \
    --language "{language}" \
    --output_dir "{dir_whisper}" \
    --device "cuda" \
    --align_model "WAV2VEC2_ASR_LARGE_LV60K_960H" \
    --diarize \
    --min_speakers "{num_speakers}" \
    --max_speakers "{num_speakers}" \
    --output_format "srt" \
    --highlight_words True \
    --hf_token "{hf_token}"
```

**Comment by sorgfresser** [comment]
Warnings are completely fine and can be ignored, they are caused by the `pyannote` version whisperX is using. They were introduced in #210 and should not be the reason for any failure. Since I am curious: if you don't specify any ouput format and dir whatsoever, do you get an `srt`?

**Comment by 7k50** [comment]
> Warnings are completely fine and can be ignored, they are caused by the `pyannote` version whisperX is using. They were introduced in #210 and should not be the reason for any failure. Since I am curious: if you don't specify any ouput format and dir whatsoever, do you get an `srt`?

Hi, thanks very much for clarifying.

I tried your suggestion of not output_format and output_dir whatsoever – but it did not help with my issue that output files aren't being created. I have created a new issue here: https://github.com/m-bain/whisperX/issues/272

**Comment by Barabazs** [comment]
Closing as resolved in #272
